# **Biomarker Discovery: Transcriptomics**
Author (@slack): Mary Dhevanayagam (@Shanu)

Biomarkers are measurable characteristics such as genes, proteins, etc. that indicate a normal or abnormal process in the body. Biomarker discovery is the process of identifying, measuring and validating biomarkers in biological samples. Some applications of biomarkers include disease diagnosis, prognosis and monitoring treatment<sup>1,2</sup>.

***Figure 1.*** The use of omics pipelines in biomarker discovery<sup>2</sup>.

## **Cancer Transcriptomics**
Transcriptomics is the study of the transcriptome, which refers to all the RNA transcripts produced from the genome<sup>3</sup>. These include mRNAs (messenger RNAs) and ncRNAs (non-coding RNAs) which have different cellular functions. Some examples of ncRNAs include rRNA (ribosomal RNA), miRNA (micro-RNA), lncRNA (long non-coding RNA), and circRNA (circular RNA)<sup>4</sup>. RNA molecules can be transcribed during normal physiological conditions or abnormal pathological conditions<sup>5</sup>. Aberrant expression of RNA molecules can be associated with carcinogenesis<sup>6</sup>. Profiling and analysing the transcriptome can provide information on gene function, regulation of gene expression and modifications of RNA transcripts<sup>5</sup>. Therefore, analysing the transcriptome enables a personalised approach in cancer biomarker discovery for the diagnosis, prognosis and treatment of cancer. 

***Figure 2.*** Different omics-based approaches to analyse gene expression<sup>5</sup>.

High-throughput technologies such as RNA sequencing (RNA-seq) and microarrays have enabled to perform transcriptome analysis in a time-efficient manner<sup>7</sup>. Differential gene expression (DGE) occurs due to genetic or epigenetic changes in the genome, and DGE analysis is often used for RNA-seq data analysis<sup>8</sup>. Transcriptome-based studies help to expand the knowledge on the molecular subtypes of cancer. RNA-seq is a widely used technology for studying aberrant gene expression associated with various malignant conditions, and so is useful for identifying new undiscovered tumour biomarkers and developing novel strategies for cancer treatment<sup>5</sup>. 

***Figure 3.*** Applications of scRNA-seq technology<sup>11</sup>.

Single-cell RNA sequencing (scRNA-seq) technology has several applications in cancer research, such as to analyse tumour heterogeneity, gene expression patterns and signalling pathways<sup>9</sup>. Unlike bulk RNA-seq data, scRNA-seq provides a more precise measurement of the transcriptome profile to study the tumour microenvironment and the clonal composition of tumours, as well as to detect alternative splicing and genetic variations such as SNPs (single nucleotide polymorphisms) and CNVs (copy number variations)<sup>10,11</sup>. Therefore, RNA-seq technology can be used to provide a detailed analysis of the transcriptome, thus facilitating early diagnosis of cancer, monitoring cancer progression, and enabling personalised medicine for cancer therapy.



**References**

1\. Ou, F.-S. *et al.* (2021) “Biomarker discovery and validation: Statistical considerations,” *Journal of thoracic oncology: official publication of the International Association for the Study of Lung Cancer*, 16(4), pp. 537–545. doi: 10.1016/j.jtho.2021.01.1616.

2\. Ng, S. *et al.* (2023) “The benefits and pitfalls of machine learning for biomarker discovery,” *Cell and tissue research*, 394(1), pp. 17–31. doi: 10.1007/s00441-023-03816-z.

3\. Yoo, B. C. *et al.* (2018) “Clinical multi-omics strategies for the effective cancer management,” *Journal of proteomics*, 188, pp. 97–106. doi: 10.1016/j.jprot.2017.08.010.

4\. Cheng, D. *et al.* (2021) “Cancer-related circular RNA: diverse biological functions,” *Cancer cell international*, 21(1). doi: 10.1186/s12935-020-01703-z.

5\. Supplitt, S. *et al.* (2021) “Current achievements and applications of transcriptomics in personalized cancer medicine,” *International journal of molecular sciences*, 22(3), p. 1422. doi: 10.3390/ijms22031422.

6\. Yang, H. D. and Nam, S. W. (2020) “Pathogenic diversity of RNA variants and RNA variation-associated factors in cancer development,” *Experimental & molecular medicine*, 52(4), pp. 582–593. doi: 10.1038/s12276-020-0429-6.

7\. Tsimberidou, A. M. *et al.* (2022) “Transcriptomics and solid tumors: The next frontier in precision cancer medicine,” *Seminars in cancer biology*, 84, pp. 50–59. doi: 10.1016/j.semcancer.2020.09.007.

8\. Rosati, D. *et al.* (2024) “Differential gene expression analysis pipelines and bioinformatic tools for the identification of specific biomarkers: A review,” *Computational and structural biotechnology journal*, 23, pp. 1154–1168. doi: 10.1016/j.csbj.2024.02.018.

9\. Chen, S. *et al.* (2023) “Application of single-cell sequencing to the research of tumor microenvironment,” *Frontiers in immunology*, 14. doi: 10.3389/fimmu.2023.1285540.

10\. Huang, D. *et al.* (2023) “Advances in single-cell RNA sequencing and its applications in cancer research,” *Journal of hematology & oncology*, 16(1). doi: 10.1186/s13045-023-01494-6.

11\. Chong, Z.-X. *et al.* (2021) “Single-cell RNA sequencing in human lung cancer: Applications, challenges, and pathway towards personalized therapy,” *Journal of the Chinese Medical Association: JCMA*, 84(6), pp. 563–576. doi: 10.1097/jcma.0000000000000535.





